### ISSUE TITLE
Add option to create broken sdists

### ISSUE THREAD

**Issue by virtuald** [robotpy/robotpy-build#204]
From @TheTripleV:

> I think all future robotpy tars should be replaced with a dummy setup.py that prints a useful error message and exits. It gets rid of all the compile error help requests. No one trying to install robotpy should end up compiling it and anyone that wants to compile it can do it from the github repo

I like it.

**Comment by auscompgeek** [comment]
This will depend on #170 I think.

**Comment by virtuald** [comment]
Apparently you can also just not upload an sdist, which I think would have a similar effect.

**Comment by TheTripleV** [comment]
I don't think so. If you don't specify a hard robotpy version, I think the dependency solver will just keep going back until it hits an sdist from many years ago
